index.html

```diff
@@ -37,5 +39,121 @@ <h3 class="head-main-sub-title">I’am a Software Developer</h3>
+        <li>Ruby</li>
+      </ul>
+      <button class="see-project-btn full-width-btn">See Project</button>
+    </div>
+    <div class="projects not-featured">
+      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
+      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="used-tools tools-bottom">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button class="see-project-btn full-width-btn">See Project</button>
+    </div>
+    <div class="projects not-featured">
+      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
+      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="used-tools tools-bottom">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button class="see-project-btn full-width-btn">See Project</button>
+    </div>
+    <div class="projects not-featured">
+      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
+      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="used-tools tools-bottom">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button class="see-project-btn full-width-btn">See Project</button>
+    </div>
+  </section>
+  <section class="about-me work-section">
+    <div class="about">
+      <h1 class="about-title">About me</h1>
+      <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="see-project-btn about-btn">Get My Resume</button>
+    </div>
+    <hr/>
+    <div class="skills">
+      <div>
+        <i class="fa-sharp fa-solid fa-diamond"></i><h3>Language</h3>
+      </div>
+      <lu class="sub-skill">
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+      </lu>
+      <div>
+      <i class="fa-sharp fa-solid fa-square"></i><h3>Frameworks</h3>
+      </div>
+      <lu class="sub-skill">
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+      </lu>
+      <div>
+      <i class="fa-sharp fa-solid fa-circle"></i><h3>Skills</h3>
+      </div>
+      <lu class="sub-skill">
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </lu>
+
```

Comment:
- Hey, please, let's fix this typo as this should be `ul` and not `lu` to improve the readability in our codebase.